packages/vega-typings/types/spec/transform.d.ts

```diff
@@ -1,6 +1,6 @@
 import { SignalRef, Compare, Vector2, ExprRef, FontWeight, FontStyle, Vector7 } from '.';
 
-export type Transform =
+export type Transforms =
```

Comment:
FYI it's not idiomatic in TypeScript to pluralize the name of a union type, because then a variable representing a single transform looks like `transform: Transforms`. A plural type name should imply that values of that type are somehow plural in nature, e.g.

```ts
type Transforms = Transform[];
```

Besides that, this seems like a needless backwards-breaking change and one made extra confusing by the fact that there is a _new_ `Transform` type, which represents something completely different than the old thing. Just my $.02!

Reply:
I see your points. The new transform type is for something that's actually a Vega property while this is just a Union type. I hope it didn't cause too much trouble and I'll be more mindful with breaking changes in the typings in the future.